### ISSUE TITLE
krane restart

### ISSUE THREAD

**Issue by dturn** [Shopify/kubernetes-deploy#530]
## Problem
Add the `krane restart` command, https://docs.google.com/document/d/1oInUsKplYGNWTymPY48xtDCx3x4Cwc-phcEKUkt2v4U/view# has details on flags that need to be supported.

This is blocked by #524.

The actual restart task doesn't match our standard interface convention. Please change  `perform/perform!` to `run/run!` as well as ensure that `new` & `run` use the same conventions as the other tasks, its fine to do this in a separate PR. 

## Possible solution
Build a thor task that calls RestartTask.new.

## Alternative solutions

Depending on the difficulty of supporting `--verify-result=false` this may need to be split into a second PR. We've discussed adding a new command that can stand alone verify a namespace, if a second PR is required please do it with an eye towards supporting such a command. 

## Acceptance criteria
- The command must work with all flags
- The command should have tests. 

## Additional context
#256 is the tracking issue for the new CLI.

**Comment by RyanBrushett** [comment]
https://github.com/Shopify/kubernetes-deploy/pull/548

**Comment by dturn** [comment]
The initial pass was added in `548. We still need to support the `-f` flag.  Thor merged https://github.com/erikhuda/thor/pull/674 (allow repeatable options), but they haven't cut a release, depending on the timeline we may point at a sha.

**Comment by dturn** [comment]
To help us prioritize work I've  cut separate issue for adding the `-f` flag. https://github.com/Shopify/kubernetes-deploy/issues/558
